modules/web/gen-default-nodes/def-to-fragment.js

```diff
@@ -0,0 +1,299 @@
+        return NodeTypes.createTopLevelNodeFragment(
+            `
+            service<fs> service1 {
+                resource echo1 (fs:FileSystemEvent m) {
+
+                }
+            }
+            `);
+    },
+    createFTPServiceDef: () => {
+        return NodeTypes.createTopLevelNodeFragment(
+            `
+            service<ftp> service1 {
+                resource echo1 (ftp:FTPServerEvent m) {
+
+                }
+            }
+            `);
+    },
+    createMainFunction: () => {
+        return NodeTypes.createTopLevelNodeFragment(`
+            function main(string[] args) {
+
+            }
+        `);
+    },
+    createFunction: () => {
+        return NodeTypes.createTopLevelNodeFragment(`
+                    function function1() {
+
+                    }
+                `);
+    },
+    createConnector: () => {
+        return NodeTypes.createTopLevelNodeFragment(`
+            connector ClientConnector(string url) {
+                action action1(){
+
+                }
+            }
+        `);
+    },
+    createConnectorAction: () => {
+        return NodeTypes.createConnectorActionFragment(`
+            action action1(){
+
+            }
+        `);
+    },
+    createHTTPResource: () => {
+        return NodeTypes.createServiceResourceFragment(`
+            resource echo1 (http:Request req, http:Response res) {
+
+            }
+        `);
+    },
+    createFSResource: () => {
+        return NodeTypes.createServiceResourceFragment(`
+            resource echo1 (fs:FileSystemEvent m) {
+
+            }
+        `);
+    },
+    createFTPResource: () => {
+        return NodeTypes.createServiceResourceFragment(`
+            resource echo1 (ftp:FTPServerEvent m) {
+
+            }
+        `);
+    },
+    createJMSResource: () => {
+        return NodeTypes.createServiceResourceFragment(`
+            resource echo1 (jms:JMSMessage request) {
+
+            }
+        `);
+    },
+    createStruct: () => {
+        return NodeTypes.createTopLevelNodeFragment(`
+            struct struct1 {
+
+            }
+        `);
+    },
+    createTransformer: () => {
+        return NodeTypes.createTopLevelNodeFragment(`
+            transformer <Source a, Target b> newTransformer (){
+
+            }
+        `);
+    },
+    createWorkerFragment: () => {
+        return NodeTypes.createWorkerFragment(`
+            worker worker1 {
+            }
+        `);
+    },
+    createDefaultWorkerFragment: () => {
+        return NodeTypes.createWorkerFragment(`
+            worker default {
+            }
+        `);
+    },
+    createAnnotation: () => {
+        return NodeTypes.createTopLevelNodeFragment(`
+            annotation annotation1 {
+            }
+        `);
+    },
+    createAssignmentStmt: () => {
+        return NodeTypes.createStatementFragment('var a = 1;');
+    },
+    createBindStmt: () => {
+        return NodeTypes.createStatementFragment('bind __connector with __endpoint;');
```

Comment:
Not related to your fix, but shall we also change this to `bind _connector with _endpoint;` ? Earlier we had a concern raised for using double underscore as variable prefix.